content/momentum/changelog/4/4-5-0.md

```diff
@@ -0,0 +1,79 @@
+|SD-2751|Story|Adjust threshold for triggering automated memory/thread stats collection in panic log
+|SD-2764|Story|HTTP endpoints to retrieve queue sizes
+|SD-2846|Story|Fix Critical "Non-constant format string" alerts in Momentum
+|SD-2847|Story|Fix Critical "Use of potentially dangerous function" alerts in Momentum
+|SD-2848|Story|Fix Critical "Unbounded write" alerts in Momentum
+|SD-2849|Story|Fix Critical "Potentially overrunning write" alerts in Momentum
+|SD-2850|Story|Fix critical "Static array access may cause overflow" alerts in Momentum
+|SD-2851|Story|Fix Critical "Potentially unsafe use of strcat" alerts in Momentum
+|SD-2855|Story|Fix High "Use of a broken or risky cryptographic algorithm" alert in Momentum
+|SD-2856|Story|Fix High "Multiplication result converted to larger type" alert
+|SD-2857|Story|Fix High "Call to alloca in a loop" alert
+|SD-2858|Story|Fix High "Potentially uninitialized local variable" alerts
+|SD-2867|Story|Fix High "Array offset used before range check" alert in Momentum
+|SD-2868|Story|Fix High  "Incorrect 'not' operator usage" alert
+|SD-2870|Story|Fix Error "Duplicate include guard" alert in Momentum
+|SD-2872|Story|Fix Warning " Unsigned comparison to zero" alert
+|SD-3031|Story|Simplify tuning of outbound connection limit on MTA nodes
+|SD-3061|Story|Update sk_tool so it works with auth login for SP
+|SD-3070|Story|Fix transmission_id truncation when calling msg_gen_check_transmission_expiration_hook
+|SD-3074|Story|Fix Warning "Lossy function result cast"
+|SD-3081|Story|Add millisecond precision to SMTP tracing
+|SD-3083|Story|Support lua files in add-local_d-file.yml playbook
+|SD-3087|Story|Momentum summary HTTP endpoint
+|SD-3093|Story|Run lua object -> json string serialization for http responses into a thread pool
+|SD-3099|Story|Fix Critical "Unbounded write" codeQL alert
+|SD-3170|Story|Momentum Transmissions API: Allow blank or empty subject header and customer supplied header values
+|SD-3206|Story|Issues with behavior when STARTTLS is issued on an in-use connection
+|SD-3231|Story|Stop encoding campaign ids in inline-seeds request
+|SD-3261|Story|Audit and cleanup "stale" CP files
+|SD-3323|Story|Security audit found ET servers running potentially exploitable nginx
+|SD-3339|Story|Standardize on transmission level substitution data size limit of 300KB in all clusters
+|SD-3368|Story|Perl test for DKIM Oversigning to Avoid Replay Attacks
+|SD-3438|Story|Documentation: Sending IP selection criteria
+|SD-3503|Story|Prevent protocol deviations related to openssl without the performance impact of SD-2734
+|SD-3558|Story|Implement better visibility into "transmission took too long" events
+|SD-3586|Story|Integrate the new store_inbound_messages internal account option into Momentum's accounts memcache
+|SD-3602|Story|Run relay_webhook build_request in a new threadpool
+|SD-3622|Story|Move JSON parsing and utf-8 validation out of CPU thread pool to dedicated thread pool
```

Comment:
Not on-prem changes but use for documentation of best practices

Reply:
Again, this is an older commit. I've since changed alot.